codex-ui/src/vue/components/alert/AlertContainer.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div
+    :class="[
+      $style.alert__container,
+      $style['alert__container--' + defaultAlertOpt.position]
```

Comment:
lets remove position property to simplify system

Reply:
removed